docs/site/content/docs/guides/microfrontends.mdx

```diff
@@ -0,0 +1,213 @@
+- **Optimized builds**: Cache and parallelize builds across all your applications
+
+## Quick start
+
+1. `npx create-turbo@latest --example with-microfrontends`
+2. Run `turbo dev` in your terminal
+3. Visit `localhost:3000` in your browser. The logs for`web#dev` task show that the `web` application's `/` route compiled.
+4. Visit `localhost:3000/docs` in your browser. The logs for the `docs#dev` task show that the `docs` application's `/` route compiled.
+
+## Setting up microfrontends in your Turborepo
+
+<Callout>
+  The tutorial below is using `npx create-turbo@latest` if you want to follow
+  along.
+</Callout>
+
+### 1. Create your applications
+
+Structure your repository with multiple applications:
+
+<Files>
+  <Folder name="apps" defaultOpen>
+    <Folder name="web" />
+    <Folder name="docs" />
+  </Folder>
+</Files>
+
+### 2. Add the configuration
+
+Create a `microfrontends.json` or `microfrontends.jsonc` in your parent application:
+
+```json title="./apps/web/microfrontends.json"
+{
+  "version": "1",
+  "applications": {
+    "docs": {}
+  }
+}
+```
+
+### 3. Configure your development tasks
+
+Ensure each microfrontend has a development task in its `package.json` that points to the port that Turborepo supplies:
+
+```json title="./apps/docs/package.json"
+{
+  "name": "docs",
+  "scripts": {
+    "dev": "next dev --port $(microfrontends port)", // [!code highlight]
+    "build": "next build"
+  }
+}
+```
+
+Refer to the documentation for your framework, library, or runtime to learn how to set the port for the script.
+
+### 4. Add the `@turbo/microfrontends` package to child apps
+
+Child applications need the `@vercel/microfrontends` package added to their `package.json` to be able to proxy their routing.
+
+<PackageManagerTabs>
+<Tab value="pnpm">
+```diff title="./apps/docs/package.json"
+{
+  "name": "docs",
+  "devDependencies": {
++   "@turbo/microfrontends": "workspace:*"
+  }
+}
+```
+</Tab>
+<Tab value="yarn">
+```diff title="./apps/docs/package.json"
+{
+  "name": "docs",
+  "devDependencies": {
++   "@turbo/microfrontends": "workspace:*"
+  }
+}
+```
+</Tab>
+<Tab value="npm">
+```diff title="./apps/docs/package.json"
+{
+  "name": "docs",
+  "devDependencies": {
++   "@turbo/microfrontends": "workspace:*"
+  }
+}
+```
+</Tab>
+<Tab value="bun (Beta)">
+```diff title="./apps/docs/package.json"
+{
+  "name": "docs",
+  "devDependencies": {
++   "@turbo/microfrontends": "workspace:*"
+  }
+}
+```
+</Tab>
+</PackageManagerTabs>
+
+### 5. Set up your turbo.json
+
+Ensure Turborepo is handling your development workflows:
+
+```json title="./turbo.json"
+{
+  "tasks": {
+    "dev": {
+      "cache": false,
+      "persistent": true
+    }
+  }
+}
+```
+
+## Running microfrontends
+
+Start all your microfrontends simultaneously:
+
+```bash title="Terminal"
+turbo dev
+```
+
+Or run specific applications:
+
+```bash title="Terminal"
+turbo dev --filter=docs
+```
```

Comment:
A callout that if only `docs` is run all `web` routes will be served by the "default" host might be useful.